packages/fluentui/react-bindings/src/FocusZone/FocusZone.tsx

```diff
@@ -79,6 +86,7 @@ export default class FocusZone extends React.Component<FocusZoneProps> implement
     direction: FocusZoneDirection.bidirectional,
     as: 'div',
     preventDefaultWhenHandled: true,
+    shouldRaiseClicks: false,
```

Comment:
can you please add it to the test as well? https://github.com/microsoft/fluentui/blob/b66a3351f9b643bc4845fe012e9a3afc42b1654f/packages/fluentui/react-bindings/test/hooks/useAccessibility-test.tsx#L332